tests/phpunit/util/wpml-pb-test-case2.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+abstract class WPML_PB_TestCase2 extends OTGS_TestCase {
```

Comment:
@adrianosferreira This is a copy of the file from pagebuilders. I think there must be something wrong with the way we are structuring the code if we need to do this.

Reply:
@brucepearson actually I don't need this, removed.